src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -144,4 +144,92 @@ void two_index_constraint(
     const tnsr::iaa<DataType, SpatialDim, Frame>&
         three_index_constraint) noexcept;
 // @}
+
+// @{
+/*!
+ * \brief Computes the generalized-harmonic F constraint.
+ *
+ * \details Computes the generalized-harmonic F constraint
```

Comment:
`generalized harmonic \f${\cal F}\f$`

Reply:
I dropped the \cal notation in all other constraints, and I want to drop it here, too. I think the cal font is nice in the paper but needlessly clutters the comments in spectre, so for consistency I'd like to continue not using the \cal font for the F constraint.